bin/build-locales

```diff
@@ -15,65 +16,70 @@ const chalk = require('chalk');
 const toSource = require('tosource');
 const oneLine = require('common-tags').oneLine;
 
-const appName = config.get('appName');
+const app = config.get('appName');
+const buildAll = process.env.ADDONS_FRONTEND_BUILD_ALL === '1';
```

Comment:
(But also a comment explaining what this does exactly would be handy. It took me a second to grok.